internal/cli/atlas/backup/exports/jobs/watch.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jobs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type WatchOpts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	id          string
+	clusterName string
+	store       store.ExportJobsDescriber
+}
+
+func (opts *WatchOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *WatchOpts) watcher() (bool, error) {
+	result, err := opts.store.ExportJob(opts.ConfigProjectID(), opts.clusterName, opts.id)
+	if err != nil {
+		return false, err
+	}
+	return result.State == "Successful" || result.State == "Failed" || result.State == "Cancelled", nil
+}
+
+func (opts *WatchOpts) Run() error {
+	if err := opts.Watch(opts.watcher); err != nil {
+		return err
+	}
+
+	return opts.Print(nil)
+}
+
+// WatchBuilder atlas backup(s) export(s) job(s) watch <exportJobId>.
+func WatchBuilder() *cobra.Command {
+	opts := new(WatchOpts)
+	cmd := &cobra.Command{
+		Use:   "watch <bucketId>",
+		Short: "Watch for an export job to complete.",
+		Long: `This command checks the export job's status periodically until it reaches a completed, cancelled or failed status. 
+Once the export reaches the expected status, the command prints "Export completed."
+If you run the command in the terminal, it blocks the terminal session until the resource status completes or fails.
+You can interrupt the command's polling at any time with CTRL-C.`,
```

Comment:
Can you add the required permission here? If any permission works, can you add "User accounts and API keys with any role can run this command."